apps/demos/Demos/Lookup/Basics/Angular/app/app.component.ts

```diff
@@ -2,8 +2,8 @@ import { NgModule, Component, enableProdMode } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { DxLookupModule } from 'devextreme-angular';
-import DataSource from 'devextreme/data/data_source';
-import ArrayStore from 'devextreme/data/array_store';
+import { DataSource } from 'devextreme-angular/common/data';
+import { ArrayStore } from 'devextreme-angular/common/data';
```

Comment:
I have just one complaint - it's cosmetic, but since these are demos, it's nevertheless important: there should not be two lines that import from the same module. In this PR, there is a number of files that violate this rule. Lint should have caught it, but for some reason it didn't.

```
// Any occurrence of

import { a } from 'foo/bar';
import { b } from 'foo/bar';

// should be refactored to

import { a, b } from 'foo/bar';
```

Looks like all such occurrences import from /common/data, so maybe if you search for `/common/data';` and notice which files have more than one result, you will quickly find where to fix.